modulefiles/wflow_gaea-c5.lua

```diff
@@ -7,14 +7,15 @@ whatis([===[Loads libraries needed for running the UFS SRW App on gaea ]===])
 
 unload("python")
 load("set_pythonpath")
-load("conda")
+prepend_path("MODULEPATH","/lustre/f2/dev/role.epic/contrib/C5/miniconda3/modulefiles")
+load(pathJoin("miniconda3", os.getenv("miniconda3_ver") or "4.12.0"))
 prepend_path("MODULEPATH","/lustre/f2/dev/role.epic/contrib/C5/rocoto/modulefiles")
 load("rocoto")
 
 pushenv("MKLROOT", "/opt/intel/oneapi/mkl/2023.1.0/")
 
 if mode() == "load" then
    LmodMsgRaw([===[Please do the following to activate conda:
-       > conda activate srw_app
+       > conda activate workflow_tools
```

Comment:
While this change will allow manual runs of the WE2E tests, the automated tests require that the `workflow_tools` conda environment be added to `ush/load_modules_wflow.sh`.  By default, all automated tests load the `srw_app` conda environment, which causes the automated tests to fail to generate due to missing `jinja2`.

Reply:
Done. Please try now.

Reply:
Thanks, @RatkoVasic-NOAA! Rerunning now.